test/OrleansTestKit.Tests/Grains/HelloTimers.cs

```diff
@@ -9,6 +9,7 @@ public class HelloTimers : Grain<HelloTimersState>, IGrainWithIntegerKey
         private IDisposable _timer0;
         private IDisposable _timer1;
         private IDisposable _timer2;
+        private IDisposable? _secretTimer;
```

Comment:
Are nullable reference types enabled on this project?

Reply:
no, just force of habbit - removed.